create_graphics.py

```diff
@@ -1,368 +1,371 @@
+def parse_args(argv: list) -> Namespace:
+    """
+    Set up argparse command line arguments, and return the Namespace
+    containing the settings.
+    """
 
-    parser = argparse.ArgumentParser(description='Script to drive the \
-                                     creation of graphices files.')
+    parser = ArgumentParser(
+        description="Script to drive the \
+                                     creation of graphices files."
+    )
 
     # Positional argument
     parser.add_argument(
-        'graphic_type',
-        choices=['maps', 'skewts', 'enspanel', 'diff'],
-        help='The type of graphics to create.',
-        )
+        "graphic_type",
+        choices=["maps", "skewts", "enspanel", "diff"],
+        help="The type of graphics to create.",
+    )
 
     # Short args
     parser.add_argument(
-        '-r',
-        dest='img_res',
+        "-r",
+        dest="img_res",
         default=72,
         required=False,
-        help='Resolution of output images in DPI. Recommended to stay below 1000. Default = 72',
+        help="Resolution of output images in DPI. Recommended to stay below 1000. Default = 72",
         type=int,
-        )
+    )
     parser.add_argument(
-        '-a',
-        dest='data_age',
+        "-a",
+        dest="data_age",
         default=3,
-        help='Age in minutes required for data files to be complete. Default = 3',
+        help="Age in minutes required for data files to be complete. Default = 3",
         type=int,
-        )
+    )
     parser.add_argument(
-        '-d',
-        dest='data_root',
-        help='Cycle-independant data directory location. Provide more than one \
+        "-d",
+        dest="data_root",
+        help="Cycle-independant data directory location. Provide more than one \
         data path if data input files should be combined. When providing \
         multiple options, the same number of options is required for the \
-        --file_tmpl flag.',
-        nargs='+',
+        --file_tmpl flag.",
+        nargs="+",
         required=True,
-        )
+        type=Path,
+    )
     parser.add_argument(
-        '-f',
-        dest='fcst_hour',
-        help='A list describing forecast hours.  If one argument, \
+        "-f",
+        dest="fcst_hour",
+        help="A list describing forecast hours.  If one argument, \
         one fhr will be processed.  If 2 or 3 arguments, a sequence \
         of forecast hours [start, stop, [increment]] will be \
         processed.  If more than 3 arguments, the list is processed \
-        as-is.',
-        nargs='+',
+        as-is.",
+        nargs="+",
         required=True,
         type=int,
-        )
+    )
     parser.add_argument(
-        '-m',
-        default='Unnamed Experiment',
-        dest='model_name',
-        help='string to use in title of graphic.',
+        "-m",
+        default="Unnamed Experiment",
+        dest="model_name",
+        help="string to use in title of graphic.",
```

Comment:
```suggestion
        help="String to use in title of graphic.",
```
For consistency with other help strings.